definitions/infra-awsrekognition/definition.yml

```diff
@@ -0,0 +1,11 @@
+domain: INFRA
+type: AWSREKOGNITION
+goldenTags:
+- aws.accountId
+- aws.awsRegion
+- aws.region
+- aws.rekognition.ProjectName
+- aws.rekognition.VersionName
+configuration:
```

Comment:
should there be a `dashboardTemplates` entry?

Reply:
@mkrichevsky  I see some of the entities have it and some not. Why do we need that entry?

Reply:
I think this question can be answered better by the repo mantainers

Reply:
w.r.t conversation with entity-pl,atform (https://newrelic.slack.com/archives/CH5MLECNN/p1684143632295339?thread_ts=1684138658.146869&cid=CH5MLECNN) adding dashboardTemplates. Thanks @mkrichevsky  for raising it